README.md

```diff
@@ -571,6 +573,46 @@ We have a small collection of algorithms, one for every day of the month. Scroll
       you       2
       ```
 
+## **December 27 - Cup and String Phone**
+  - **Problem :**
+    - Tharun is playing with his friends using a string phone. To make this interesting he asked his friends to join him. He asked them to bring cup of their own and he will bring the strings for all of them so that all can communicate with each other. Now he wants to find the number of strings required to communicate with his friends. Help him with a program that takes a number of friends as input and give the number of strings as output.
+
+    For example Tharun plays with 2 of his friends. Then he will bring 3 Strings.
+
+  - **Example :**
+  
+    Input : 7
+    Output : 28
+ 
+  - **Uses :**
+    - Triangular Series
+    - Networking between computers
+    - No. of greetings exchanged among 'n' people
+    - No. of matches played by 'n' teams
+
+  - **Resources :**
+    - [Triangular Series](https://en.wikipedia.org/wiki/Triangular_number)
+
+
+## **December 28 - Number of Diagonals**
+  - **Problem :**
+    - There is a n-sided polygon. Find the number of diagnols that are possible. Do not count duplicates lines.
```

Comment:
Explain how you find the number of diagonals. Use a rectangle & pentagon as an example.